.circleci/config.yml

```diff
@@ -1,15 +1,15 @@
 version: 2.1
 orbs:
-  samvera: samvera/circleci-orb@0.3.2
+  samvera: samvera/circleci-orb@1
 jobs:
   bundle_and_test:
     parameters:
       ruby_version:
         type: string
-        default: 2.6.3
+        default: 2.7.5
       rails_version:
         type: string
-        default: 5.2.3
+        default: 5.2.6
```

Comment:
If you're testing on ruby 6.1 and even 7 now, do you still want to make the "default" "main" test up here 5.2?  Or should swe swap em?